JapaneseFilter/sections/css_extended.txt

```diff
@@ -17,6 +17,7 @@ lifehacker.jp#?#.lh-slideInner > li:has(> .lh-summary > .lh-summary-meta > .lh-s
 7narabe.net,blog-rpg.com,cooklog.net,cosplay-navi.com,gjgd.net,guhaw.com,kuizu.net,mangalog.com,or-hell.com,ria10.com,sugo-roku.com,tou3.com,zoku-sei.com#?#.pad #ninja-blog-inactive:upward(.pad)
 7narabe.net,sakeblog.net,yotsumeyui.com#?##e-zone > h2 a[href=""]:upward(h2)
 7narabe.net,sakeblog.net,yotsumeyui.com#?#.ebody #ninja-blog-inactive:upward(.ebody)
+anime-sharing.com#?##sidebar > li:has(> div.block .blocksubhead > a#collapse_block_html_4)
```

Comment:
why not
`anime-sharing.com#?##sidebar > li:has(> div.block > div.blocksubhead > a#collapse_block_html_4)`